src/renderers/shared/ReactDOMFrameScheduling.js

```diff
@@ -23,10 +23,21 @@
 import type {Deadline} from 'ReactFiberReconciler';
 
 var invariant = require('fbjs/lib/invariant');
+var ExecutionEnvironment = require('fbjs/lib/ExecutionEnvironment');
 
 // TODO: There's no way to cancel these, because Fiber doesn't atm.
 let rAF: (callback: (time: number) => void) => number;
 let rIC: (callback: (deadline: Deadline) => void) => number;
+
+if (!ExecutionEnvironment.canUseDOM) {
```

Comment:
There's a [`window.addEventListener`](https://github.com/prometheansacrifice/react/blob/ce00e20b61b887fe6f6b9b966f00929a07cc8c27/src/renderers/shared/ReactDOMFrameScheduling.js#L92) too. Use the same trick there?